### ISSUE TITLE
Support `dnxcore50` framework

### ISSUE THREAD

**Issue by saxx** [aelij/AsyncFriendlyStackTrace#5]
I'm still using RC1 of .NET Core and I want to wait to upgrade to the latest RC2 bits until it's finally released. 

As far as I understand, with RC2 the frameworks will completely change (.NET Platform Standard), but for now I still work with `dnxcore50` framework in my apps for .NET Core.

Unfortunately the AsyncFriendlyStackTrace NuGet package only support `net451` and `dotnet5.4` frameworks at the moment. So I cannot reference it from my `dnxcore50` project.

Would it be possible to add `dnxcore50` to the supported frameworks? I know that this would only be temporary until the .NET Platform Standard finally arrives, but it would help in my case until I can move on from RC1.

**Comment by aelij** [comment]
Sure, you're welcome to open submit a PR.

**Comment by saxx** [comment]
I tried to submit a quick PR, but I cannot even get the source to compile on my machine, because it shows some weird errors with dotnet5.4 (certain dependencies for dotnet5.4 are not available). What do I have to install to get it to compile?

**Comment by aelij** [comment]
I've added it (also pushed to NuGet).

**Comment by saxx** [comment]
Thank you for the quick reply. Unfortunately the new package does not work on my machine:

![capture](https://cloud.githubusercontent.com/assets/1190606/13595064/e5844c6a-e508-11e5-9d27-1fd4dd6c940f.PNG)

I think this is because the referenced `Microsoft.CSharp` dependency is too new for RC1. 

I'm not even sure that dependency is needed. For now I included the AsyncFriendlyStackTrace sources directly in my project (just to try it out on `dnxcore50`), and I only needed to add a dependency to `"System.Diagnostics.StackTrace": "4.0.1-beta-23516"` to get it to compile in `dnxcore50`.
